public/locales/fr/common.json

```
@@ -9,19 +9,19 @@
   "albumArtist": "Artiste de l'album",
   "history": "Historique",
   "yourProfile": "Votre profil",
-  "yourHistory": "Your history",
-  "scrobbleManually": "Scrobble manually",
-  "scrobbleFromOtherUser": "Scrobble from other user",
-  "otherUser": "User",
+  "yourHistory": "Votre historique",
+  "scrobbleManually": "Scrobbler manuellement",
+  "scrobbleFromOtherUser": "Scrobbler depuis un autre utilisateur",
+  "otherUser": "Utilisateur",
   "or": "ou",
   "refresh": "Actualiser",
-  "findFriendCopy": "Enter a last.fm username to see their last tracks",
-  "visitProfile": "View profile",
-  "search": "Search",
-  "backToSearch": "Back to search",
-  "userNotFound": "User not found",
-  "invalidUsername": "Invalid username",
-  "recentlySearchedUsers": "Searched recently",
+  "findFriendCopy": "Entrez un nom d'utilisateur last.fm pour voir leurs dernières pistes",
+  "visitProfile": "Voir le profil",
+  "search": "Rechercher",
+  "backToSearch": "Retourner sur la recherche",
+  "userNotFound": "Aucun utilisateur trouvé",
+  "invalidUsername": "Nom d'utilisation invalide",
```

Shouldn't this be "Nom d'utilisateur invalide"?

```suggestion
  "invalidUsername": "Nom d'utilisateur invalide",
```

---

Nom d'utilisateur invalide = Invalid username, not 'utilisation' which translates into `use` or also `usage`.

---

ok, it should be `utilisateur` then! 👍

---

(Can you accept the suggested code change here? Thanks!)